PWGHF/D2H/TableProducer/dataCreatorCharmHadPiReduced.cxx

```diff
@@ -741,21 +914,27 @@ struct HfDataCreatorCharmHadPiReduced {
         fillHfCandCharm = true;
       }                                                           // pion loop
       if (fillHfCandCharm) {                                      // fill candCplus table only once per D candidate
-        if constexpr (decChannel == DecayChannel::B0ToDminusPi) { // D∓ → π∓ K± π∓
+        if constexpr (decChannel == DecayChannel::B0ToDminusPi || decChannel == DecayChannel::BsToDsminusPi) { // D∓ → π∓ K± π∓ and Ds∓ → K∓ K± π∓
           hfCand3Prong(charmHadDauTracks[0].globalIndex(), charmHadDauTracks[1].globalIndex(), charmHadDauTracks[2].globalIndex(),
                        indexHfReducedCollision,
                        trackParCovCharmHad.getX(), trackParCovCharmHad.getAlpha(),
                        trackParCovCharmHad.getY(), trackParCovCharmHad.getZ(), trackParCovCharmHad.getSnp(),
                        trackParCovCharmHad.getTgl(), trackParCovCharmHad.getQ2Pt(),
                        candC.xSecondaryVertex(), candC.ySecondaryVertex(), candC.zSecondaryVertex(), invMassC0);
+          // TODO: add invMassC1 to the table to support Bs -> Ds Pi
```

Comment:
why not done now already?

Reply:
This requires to change the data model also for other analyses, so as a first step I tried to implement all the less invasive parts. I'm looking into it now

Reply:
It seems to me that including it now should anyway not disrupt the rest of the workflows, since if it is not used later on it is not a problem, but I might have missed something

Reply:
I had to modify some tables and I touched a few other workflows with minor changes. You can see the changes in the new commits